### ISSUE TITLE
ListBoxItem HTML markup is invalid and not accessible

### ISSUE THREAD

**Issue by is-sematell** [skeletonlabs/skeleton#2780]
### Current Behavior

The HTML code generated by ListBox produces invalid HTML and does not meet WCAG 2.0 parsing compliance for accessibility. E.g.
```html
<label>
   <div class="listbox-item cursor-pointer -outline-offset-[3px]  rounded-token px-4 py-2 variant-filled " data-testid="listbox-item" role="option" aria-selected="true" tabindex="0">
      <div class="h-0 w-0 overflow-hidden"><input type="radio" name="medium" value="books" tabindex="-1"></div>
      <div class="listbox-label flex items-center space-x-4">
         <div class="listbox-label-lead  "><i class="fa-solid fa-book text-xs w-6 text-center"></i></div>
         <div class="listbox-label-content flex-1 ">Books</div>
      </div>
   </div>
</label>
```

### Expected Behavior

ListBoxItem should produce valid HTML and meet WCAG 2.0 parsing compliance.

### Steps To Reproduce

1. Use the ListBoxItem component
2. Run the [Nu Html Checker](https://validator.w3.org/nu/) on the resulting code - [bookmarklets are also available](https://validator.w3.org/nu/about.html)
3. See errors generated

### More Information

The issue can be seen when validating the ListBox documentation page: https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.skeleton.dev%2Fcomponents%2Flistboxes

**Comment by endigo9740** [comment]
Hey @is-sematell, 

We understand there are some number of accessibility issues with Skeleton v2.x. But this has has been right at the forefront for us in our next major update, which refactors and overhauls our full suite of components:

- https://github.com/skeletonlabs/skeleton/discussions/2681

i wanted to respond to this a few days ago but wanted to wait until our new announcement went live. But I believe our upcoming integration with Zag.js should address many issues like this. You can read more about what this entails here:

- https://github.com/skeletonlabs/skeleton/discussions/2784

For now, the team and I have [shifted our priority](https://github.com/skeletonlabs/skeleton/discussions/2729) from Skeleton v2 -> v3. As such we hope to deliver this as soon as possible. However, we will likely not be able to provide any turnkey solutions in the interim for the current production version.

That said, we are still open to third party contributions. If you, or any member of the community wishes to jump in an assist with this. We'll be more than welcome to this. However, I do foresee some challenges with notable structure changes. These might fall into the category of breaking changes, which we limit to major release. As such, the long term solution will be Skeleton v3.

I'll leave this up to you if you wish to keep this issue open for now, or if you would prefer to check back when v3 drops in the future.

**Comment by is-sematell** [comment]
Thanks for this reply, I appreciate it. If this is resolved in the next version of Skeleton, then there's no need to keep this and #2779 open. I'm closing it.
